### PR TITLE
Start ``simGCN.c``

### PR DESCRIPTION
**Pull request by Yanis42**
thanks to cadmic for figuring out the data stuff

### CODE REVIEW

File: include/simGCN.h [hunk 1]
```diff
@@ -49,11 +49,61 @@ typedef struct __anon_0xDB69 {
...
+    S_M_CARD_SV11 = 44,
+    S_M_CARD_SV12 = 45,
+    S_M_CARD_SV_SHARE = 46,
+    S_M_CARD_DEFAULT_ERROR = 47,
+} __anon_0x61D7;
+
 extern s32 gDVDResetToggle;
 extern s32 gbDisplayedError;
 
-extern System* gpSystem;
```
Comment (cadmic) [inline include/simGCN.h:55]: why remove this declaration?

File: src/simGCN.c [hunk 2]
```diff
@@ -0,0 +1,346 @@
+#include "simGCN.h"
+
+// TODO: move to macros.h
+#define ALIGNAS(N) __attribute__((aligned(N)))
```
Comment (cadmic) [inline src/simGCN.c:4]: sorry for making you clean up my mess, but could you move this now? It seems like will probably be useful for other files soon

File: src/simGCN.c [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+
+s32 gHighlightChoice = 0x00000001;
+__anon_0x61D7 simulatorMessageCurrent = S_M_NONE;
+s32 gResetBeginFlag = 0x00000001;
+
+char D_80134D9C[8] = "yes.raw";
+char D_80134DA4[7] = "no.raw";
+
+// TODO: struct
+static s32 gpCode;
```
Comment (cadmic) [inline src/simGCN.c:227]: should we make this struct now? (__anon_0x57A1)

File: include/simGCN.h (hunk 1 above)
Comment (Yanis42) [reply include/simGCN.h:55]: because the debug data says it's declared in simGCN.c so it doesn't make sense to extern it in this header

Comment (Yanis42) [reply include/simGCN.h:55]: maybe I misread

File: src/simGCN.c (hunk 3 above)
Comment (Yanis42) [reply src/simGCN.c:227]: I named it "Code", we can rename it later if it turns out it's a bad name

Comment (cadmic) [reply src/simGCN.c]: Yeah I think that is too generic (I would have left it anonymous myself) but I was mostly concerned about getting the struct layout right when matching functions

**Pull request merged**